project-board-crew-logic/src/main/java/project/board/crew/logic/structure/user/Users.java

```diff
@@ -0,0 +1,23 @@
+package project.board.crew.logic.structure.user;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Users {
+
+    @Id
+    private Long userId;
```

Comment:
UserId to id pobierane z zewnętrznego modułu. Co za tym idzie, powinno mieć id i userId które jest odnośnikiem do obiektu z zewnętrznego serwisu. Wyobraź sobie sytuację w której ktoś jest dodany do dwóch crew - wtedy utworzyłoby to 2 obiekty user o takim samym id i Maślak nie byłby z tego dumny. Id powinno byc automatycznie generowane